spec/src/main/asciidoc/jakarta-stl.adoc

```diff
@@ -1873,12 +1873,12 @@ doing this.
 
 ==== Deferred Values
 
-As of JSP 2.1, the new unified Expression
+The unified Expression
 Language supports the concept of deferred expressions (using the #{}
```

Comment:
Removed the old specification reference, I think it reads fine how I have it now but open to your opinion.

Reply:
Seems absolutely fine. I think most specs (at least the ones I'm looking at) did it that way.